simple_app.wsgi

```diff
@@ -1,6 +1,3 @@
 #/usr/bin/python
-import sys
-sys.path.insert(0, "simple_root/simple_app")
-sys.path.insert(0, "simple_root")
 from simple_app import create_app
 application = create_app()
```

Comment:
Glad you worked out a way around the simple_root link. That was a very weird setup.